contracts/ProjectsLibrary.sol

```diff
@@ -21,30 +21,30 @@ library ProjectsLibrary {
         projectStore.setStringValue(sha3('projectTitle', projectIndex), title);
         projectStore.setStringValue(sha3('projectDescription', projectIndex), description);
         projectStore.setAddressValue(sha3('projectOwner', projectIndex),msg.sender);
-        projectStore.setIntValue(sha3('projectBudet', projectIndex), budget);
+        projectStore.setIntValue(sha3('projectBudget', projectIndex), budget);
         projectStore.setIntValue(sha3('projectDateCreated', projectIndex), now);
         projectStore.setIntValue(sha3('projectStatus', projectIndex), 0);
     }
 
-    function getProject(address projectStoreAddress, uint projectID) constant returns (string projectTitle,
-                        string projectDescription, address projectOwner, uint projectBudet, uint projectMemberCount,
+    function getProject(address projectStoreAddress, uint projectIndex) constant returns (string projectTitle,
```

Comment:
We should remove the string returns from here as we cannot return dynamic sized data from one contract to another. The same has to be updated in Organisation.sol.